aspnet/security/anti-request-forgery.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Preventing Cross-Site Request Forgery (XSRF/CSRF) Attacks in ASP.NET Core
+author: rick-anderson
+ms.author: riande
+manager: wpickett
+ms.date: 10/14/2016
+ms.topic: article
+ms.assetid: 09607e8b-a977-4807-81aa-7a5d73bfbacd
+ms.prod: aspnet-core
+uid: security/anti-request-forgery
+---
+
+  # Preventing Cross-Site Request Forgery (XSRF/CSRF) Attacks in ASP.NET Core
+
+[Fiyaz Hasan](https://twitter.com/FiyazBinHasan), David Paquette
+
+See this [word document](https://www.dropbox.com/sh/jfmncpp0z79m9mt/AAAAB98zC4J7f9g_l6egU8nta?dl=0) and [Evolving ASP.NET Apps–Cookie Authentication](https://blogs.msdn.microsoft.com/cdndevs/2015/02/18/evolving-asp-net-appscookie-authentication/)
+
+David Paquette gave me permission to copy his blog if we add him to the authors list.
```

Comment:
@ardalis  look on the files tab, that's the first article you see. Need to delete it.